pom.xml

```diff
@@ -66,9 +66,9 @@
         <scala.parsers.version>1.0.6</scala.parsers.version>
 
         <geoserver.version>2.12.0</geoserver.version>
-        <gt.version>18-beta</gt.version>
+        <gt.version>18-RC1</gt.version>
         <jaiext.version>1.0.15</jaiext.version>
-        <imageio-ext.version>1.1.15</imageio-ext.version>
+        <imageio-ext.version>1.1.17</imageio-ext.version>
```

Comment:
does this change cqs?

Reply:
it claims not to have when i ran calc cqs